### PR TITLE
MAINT: Remove unnecessary imports

### PR DESCRIPTION
**Pull request by srinivasreddy**

### CODE REVIEW

**Comment by charris** [comment]
Looks good to me, but ... Could you redo the commit message with more information as to why the import are no longer necessary, i.e.,  `set` is now a builtin, and make the first line `MAINT: Remove unnecessary imports`.

**Comment by srinivasreddy** [comment]
Sure

**Comment by srinivasreddy** [comment]
Done

File: pavement.py [hunk 1]
```diff
@@ -65,11 +65,9 @@
...
 import subprocess
 import re
-try:
-    from hashlib import md5
-    from hashlib import sha256
-except ImportError:
-    from md5 import md5
+from hashlib import md5
+from hashlib import sha256
+
```
Comment (eric-wieser) [inline pavement.py]: Extra newline. Also, is something further down checking to see if sha256 was imported here?

Comment (srinivasreddy) [reply pavement.py]: Nope.

**Comment by srinivasreddy** [comment]
@eric-wieser  Done.

**Comment by eric-wieser** [comment]
See charris's comment about improving the message

**Comment by srinivasreddy** [comment]
@eric-wieser  Hope the current commit message is fine.

**Comment by charris** [comment]
Merging, appveyor is being flaky.

**Comment by charris** [comment]
Thanks @srinivasreddy .

**Pull request merged**